forecast-hub-report.Rmd

```diff
@@ -193,7 +194,7 @@ Overview of all the models and metrics
 
 ``` {r scoretable, fig.width = 12, results = 'asis', eval = TRUE}
 # overall summary
-plot_score_table_helper <- function(data, type) {
+plot_score_table_helper <- function(data, type, filter) {
```

Comment:
what does this do?

Reply:
:( I think this was from me looking through all of the stuff looking for what to change.